framework/wazuh/rbac/orm.py

```diff
@@ -1696,6 +1777,39 @@ def update_policy(self, policy_id: int, name: str, policy: dict,
             self.session.rollback()
             return SecurityError.ALREADY_EXIST
 
+    def migrate_data(self, manager, source: str, target: str, from_id: Optional[int] = None,
+                     to_id: Optional[int] = None):
```

Comment:
Please add the docstring

```suggestion
    def migrate_data(self, manager, source: str, target: str, from_id: Optional[int] = None,
                     to_id: Optional[int] = None) -> None:
```

Reply:
Fixed in 402c4224dd39f3a6dd3cefca2e48f78cd7c872a2.